.circleci/fetch_geth_parity_solc.sh

```diff
@@ -29,6 +29,11 @@ if [[ ! -x ${PARITY_PATH} ]]; then
 fi
 ln -sf ${PARITY_PATH} ${LOCAL_BASE}/bin/parity
 
+# Only deal with solc for Linux since it's only used for testing
+if [[ "{OS_NAME}" != "LINUX" ]]; then
```

Comment:
`{OS_NAME}` -> `${OS_NAME}`

(And you don't need quotes inside `[[ ]]`)

Reply:
Ah tnanks @ulope Was about to look why it failed. Forgot the $